omod/src/main/webapp/resources/htmlforms/mexico/consult.xml

```diff
@@ -34,11 +34,15 @@
         }
 
         .field-error {
-            color: #ff6666;
+            color: #ffc0b5;
```

Comment:
Is this the fix for the issue making abnormal values green?  If this gets the job done, that's fine, but I've got no problem with making a change to how the highlighting is done in HFE directly... it seems like what is currently happening is wrong, and I don't think there was any particular thought/strategy behind it, likely just something whoever was developing the initial ticket thought was nice to add.

Reply:
This one just makes the red a little less aggressive.